bin/setup

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+# ========== Install mysql ==========
+echo "Checking MySQL"
+
+# Check for existing mysql
+mysql_exists=`brew list -1 | grep mysql`
+if [ -z $mysql_exists ]; then
+    echo "  Installing MySQL"
+    brew install mysql
+fi
+# Check to see if mysql is running
+mysql_running=`ps ax | grep mysqld_safe | grep -v grep | awk '{print $1}'`
+if [ -z $mysql_running ]; then
+    echo "  Attempting to start MySQL"
+    brew services start mysql
+fi
+
+# Check to see if testing db is present
+mltshp_testing_db_exists=`mysql -u root -e "SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = 'mltshp_testing'" | tail -1`
+if [ -z $mltshp_testing_db_exists ]; then
+    echo "  Creating 'mltshp_testing' database"
+    mysql -u root -e "CREATE DATABASE mltshp_testing"
+fi
+
+# Check to see if development db is present
+mltshp_dev_db_exists=`mysql -u root -e "SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = 'mltshp'" | tail -1`
+if [ -z $mltshp_dev_db_exists ]; then
+    echo "  Creating and seeding 'mltshp' database"
+    mysql -u root -e "CREATE DATABASE mltshp"
+    mysql -u root -D mltshp < setup/db-install.sql
+    mysql -u root -D mltshp < setup/db-fixtures.sql
+fi
+
+# ========== Install ruby ==========
+echo "Checking Ruby install"
+
+# Regexplanation: `\[ruby-\]([^\[]+)\[.*\]` finds 2.4 from the string `'[ruby-]2.4[.1]`
+latest_ruby=`rvm list known | grep "\[ruby-\]" | tail -1 | sed -E 's/\[ruby-\]([^\[]+)\[.*\]/\1/g'`
```

Comment:
would it be possible to do this without rvm?  it's a pretty nasty tool in my experience when compared with ruby-build/chruby or rbenv -- it does horrible things in the global shell environment, to such a degree that $WORK have banned its use.

Possibly just doing a ruby --version check would be enough to avoid it.